packages/excavator-script/src/projects/designerSweatpants.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @author horrible-little-slime
+ * Track sweatiness of various sweatpants zones
+ */
+import { currentRound, equippedItem, myLocation } from "kolmafia";
+import { $item, $slot } from "libram";
+
+import { ExcavatorProject } from "../type";
+
+type SweatpantsData = { location: number; sweat: number };
+const SWEAT_PATTERN = /You get (\d)% sweatier/;
+function spadeSweatpants(
+  _encounter: string,
+  page: string,
+): SweatpantsData | null {
+  if (currentRound() !== 0) return null;
+  if (equippedItem($slot`pants`) !== $item`designer sweatpants`) return null;
+  const result = page.match(SWEAT_PATTERN);
+  if (result) return { location: myLocation().id, sweat: Number(result[1]) };
```

Comment:
how about updating toNormalizedString to
```
export function toNormalisedString(thing: Item | Location | Monster) {
  if (thing instanceof Location) {
    return `[${thing.id}]${thing.toString()}`;
  }
  return `[${thing.id}]${thing.name}`;
}
```
and use that string instead of location.id, i think it would be more readable